City-of-Helsinki/helsinki-paatokset
# PP-278: Add confidential tag to cases

**To test**
- Checkout branch, run `make drush-cr drush-cim`
- If you don't have AHJO data yet (specifically cases), run `make ahjo-migrations`
- If you do have cases, run `drush ahjo-proxy:update-cases --logic=publicity --limit=300 -v` (adjust the limit parameter or re-run the command to update all of them)
- Check cases that have a confidential ("Salassa pidettävä") publicity class: https://helsinki-paatokset.docker.so/fi/admin/content/cases?order=field_publicity_class&sort=desc (If all your cases are public, run `drush ap:update cases HEL-2021-012646 -v` to get one that is not public)
- The confidential cases should have a red "Confidential" tag next to the classification tag.
- The "Confidential" label should be translatable through UI translations
